### ISSUE TITLE
Poor Test accuracy on custom dataset 

### ISSUE THREAD

**Issue by basit123456** [TJCVRS/CRNN_Tensorflow#81]
i trained a model on a custom numberplates dataset for 80000 epochs. the train accuracy was almost 95% and training loss was around 0.4-0.6 but at test time the predict output is missing some characters but predicting rest of numberplate correctly

**Comment by TJCVRS** [comment]
@basit123456 Maybe you should check your label file or the dict json file

**Comment by basit123456** [comment]
There are 4 kind of number plates, for 3 plates the predictions are correct for 1 model predicts the numbers correct but miss some of the characters i checked the labels the true labels are correct. here are some examples of predictions 

Predict 197fd4b4-9214-4b90-b6ea-410265979612.jpg image with gt label: wdh11312 **** predict label: wh11312
Predict f5478a6f-262c-4155-b505-185253e5c8e9.jpg image with gt label: ddt52767 **** predict label: d52967
Predict b3d6e31a-f3e5-4594-af09-5851b9007570.jpg image with gt label: xax41690 **** predict label: x41690
Predict 917342a3-be9a-4c0e-a2fe-159eda7755f9.jpg image with gt label: jzl53922 **** predict label: j53913
Predict 4d72847e-7d19-4faa-80d2-0c5357298ec7.jpg image with gt label: qis06820 **** predict label: qs06820

**Comment by TJCVRS** [comment]
@basit123456 Since I don't know about your dataset I can not figure out what's happening here. But I have used this model on captcha recognition mission which is similar to your mission in my opinion and there is no such problem
